lib/ansible/plugins/strategy/free.py

```diff
@@ -172,10 +172,14 @@ def run(self, iterator, play_context):
                 if last_host == starting_host:
                     break
 
+            display.debug('_process_pending_results start')
```

Comment:
instead of surrounding the method call, put the debug in the method itself and it won't be 'strategy plugin dependent'